src/lib/core/fiber.c

```diff
@@ -1102,6 +1104,30 @@ fiber_set_name(struct fiber *fiber, const char *name)
 	fiber->name[size] = 0;
 }
 
+const char *
+fiber_get_name(struct fiber *fiber)
```

Comment:
AFAIU, `fiber_set_name_n()` is introduced to allow a zero byte inside the name. If it is so, than the `fiber_get_name()` function should accept `uint32_t *len` output parameter. Or don't accept the length parameter in both.

BTW, there is `fiber_name()` named without `get`. I would expose it into the public API under the name without `get`. AFAIR, tarantool's code generally avoid `get` for accessors.

Reply:
The same for `fiber_get_id()` vs `fiber_id()` and `fiber_get_csw()` vs `fiber_csw()`.

Reply:
1.) Actually it's introduced to avoid extra allocations, because for example rust's strings aren't nul-terminated by default. fiber_set_name adds the nul byte anyways and doesn't store the length, so, firstly, there's no need to explicitly return the length, and secondly it would mean doing extra work for computing this length.

2.) I decided to introduce another function in addition to fiber_name, because fiber_name is currently `static inline` and if I export it to ffi, it would no longer be auto inlinable, which may affect the performance. Altough I guess I could just rename the `static inline` version instead

Reply:
`fiber_name()` is used just in logs, in exceptions, and in Lua C. Generally, if you need a fiber name, then you are not writing a hot performant code block. I suggest to uninline `fiber_name()`. It will change nothing when it comes to perf.

Reply:
Removed `_get` from getter functions